src/nprintml/learn/step.py

```diff
@@ -65,8 +67,46 @@ def __init__(self, parser):
             metavar='INTEGER',
             type=int,
             help="maximum time (seconds) over which to train each model "
-                 f"(default: {AutoML.TIME_LIMIT})",
+                 "(default: %(default)s)",
+        )
+
+        # below %(prog)s will refer to subcommand -- not what we want
+        base_program = 'python -m nprintml' if parser.prog.endswith('.py') else parser.prog
+
+        learn_only_parser = parser.subparsers.add_parser(
+            'learn',
+            description="run only the AutoML step given previously-saved or "
+                        "separately-prepared feature data\n\n"
+                        "arguments related to AutoML are optionally inherited from the base "
+                        "command (e.g.: --metric):\n\n"
+                        f"    {base_program} --metric=f1 learn ./data/features.fhr",
+            formatter_class=argparse_formatter.FlexiFormatter,
+            help="run only AutoML",
+            satisfies=self.__requires__,
```

Comment:
`Learn` is only responsible for defining its subcommand and communicating to the framework what requirements are specially satisfied by invocation of this subcommand.

The subparser class and the pipeline do the rest.